getnacelle/nacelle-js
# fix: optimistic UI for qty update

### WHY are these changes introduced?

Fixes [[ENG-7269](https://nacelle.atlassian.net/browse/ENG-7269)] 



### WHAT is this pull request doing?

Fixes optimistic UI for quantity update

### How to Test
- `git checkout ENG-7269--nuxt-starter-cart-quantity`
- `cd starter/nuxt`
- `npm i && npm run dev`
- see if cart/checkout works
